app/views/user_mailer/batch_subscription_notification.text.erb

```diff
@@ -3,29 +3,29 @@
 <% @seen_summary = {} %>
 <% @creations.each_with_index do |creation, index| %>
 <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> posted a <% if creation.is_a?(Work) then %><%= this_work.backdate ? "backdated" : "new" %> work:<% else %>new chapter of <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
+<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> <%= t 'user_mailer.batch_subscription_notification.posted_a' %> <% if creation.is_a?(Work) then %><%= if this_work.backdate then t 'user_mailer.batch_subscription_notification.backdated' else t 'user_mailer.batch_subscription_notification.new' end %> <%= t 'user_mailer.batch_subscription_notification.work' %> <% else %> <%= t 'user_mailer.batch_subscription_notification.new_chapter_of' %> <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
```

Comment:
I hate the way it is all on one line but I think that is to do with it being a txt template, blank lines will have a tendency to come out on the output. I will try.